catalog/ui/catalog-ui-search/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
+log4j.rootLogger=ERROR, stdout
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
```

Comment:
What is using the log4j props file?

Reply:
Spark's servlet process. It discovers this resource file and greatly alleviates test debugging / troubleshooting pains since the test process itself doesn't have access to stacktraces. With this file present, if an NPE occurs in the web server, we get error logging on `stdout`.